src/index.html

```diff
@@ -1,14 +1,39 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <video src="styles/bgvideo.mp4"
+    autoplay
+    muted
+    loop
+    class="bgVideo"
+    >
+    </video>
+    <div class="watch">
+      <div class="watch__center"></div>
+      <div class="watch__hand watch__hand--minute"></div>
+      <div class="watch__hand watch__hand--second"></div>
+      <ul>
+        <li class="watch__hour">1</li>
```

Comment:
this is not a list of some items. screenreader can misunderstand what you want to show "him" =)